exercises/generative-models.md

```diff
@@ -146,16 +149,15 @@ Infer({method: "forward", samples: 1000}, function() {
 If we wanted to represent the diseases of many patients we might have tried to make each disease and symptom into a function from a person to whether they have that disease, like this:
 
 ~~~~
-var allergies = function(person) {return flip(.3)};
-var cold = function(person) {return flip(.2)}
+var allergies = mem(function(person) { return flip(.3) })
```

Comment:
i don't think this should have the mem... because that's the point of the final question below. If unclear you could change around the questions?